drivers/sensor/max31790/max31790_fan_fault.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 SILA Embedded Solutions GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_MAX31790_MAX31790_FAN_FAULT_H_
+#define ZEPHYR_DRIVERS_SENSOR_MAX31790_MAX31790_FAN_FAULT_H_
+
+#include <zephyr/drivers/i2c.h>
+
+struct max31790_fan_fault_config {
+	struct i2c_dt_spec i2c;
+};
+
+struct max31790_fan_fault_data {
```

Comment:
Why are these structures defined in a header file? Are they intended to be used outside of the max31790_fan_fault driver?

Reply:
It is not in the include directory, therefore only accessible locally. And it is done like this for most other sensors, therefore I went for consistency within the sensor drivers.